### PR TITLE
Fix up product metadata (and spelling)

### PR DESCRIPTION
**Pull request by gewarren**
- dotnet-core is child of dotnet so don't need to list both
- remove 'windows' where it's not showing Windows dev technologies
- change dotnet-core to aspnet-core where applicable

Contributes to https://github.com/dotnet/docs/issues/35970

Also fixes various spelling mistakes and miscellaneous.

### CODE REVIEW

File: windowsforms/Conway's-Game-of-Life/VB/readme.md [hunk 1]
```diff
@@ -49,6 +49,6 @@ The universe of the Game of Life is an infinite two-dimensional orthogonal grid
 3. Any live cell with more than three live neighbors dies, as if by overpopulation.
 4. Any dead cell with exactly three live neighbors becomes a live cell, as if by reproduction.
 
-The initial pattern constitutes the seed of the system. The first generation is created by applying the above rules simultaneously to every cell in the seed�births and deaths occur simultaneously, and the discrete moment at which this happens is sometimes called a tick (in other words, each generation is a pure function of the preceding one). The rules continue to be applied repeatedly to create further generations.
+The initial pattern constitutes the seed of the system. The first generation is created by applying the above rules simultaneously to every cell in the seed�births and deaths occur simultaneously, and the discrete moment at which this happens is sometimes called a tick (in other words, each generation is a pure function of the preceding one). The rules continue to be applied repeatedly to create further generations.
```
Comment (Tanya-Solyanik) [inline windowsforms/Conway's-Game-of-Life/VB/readme.md:52]: what character is it?

**Review by Tanya-Solyanik — APPROVED** [review]
change s related to WInForms look good.

File: windowsforms/Conway's-Game-of-Life/VB/readme.md (hunk 1 above)
Comment (gewarren) [inline windowsforms/Conway's-Game-of-Life/VB/readme.md:52]:
```suggestion
The initial pattern constitutes the seed of the system. The first generation is created by applying the previous rules simultaneously to every cell in the seed. Births and deaths occur simultaneously, and the discrete moment at which this happens is sometimes called a tick (in other words, each generation is a pure function of the preceding one). The rules continue to be applied repeatedly to create further generations.
```

Comment (gewarren) [reply windowsforms/Conway's-Game-of-Life/VB/readme.md]: I removed it - thanks.

**Pull request merged**